### ISSUE TITLE
Unknown reason for Crash

### ISSUE THREAD

**Issue by M1ntGr33n** [opentoonz/opentoonz#3289]
Opentoonz Crashes on random. There is no freeze, no "Opentoonz has stopped working" window, nothing, it only closes. There seams to be no clear trigger. The only thing that appears to be constant is that it only crashes when using the viewport (Drawing something, selecting something, undoing something etc.), it never crasehed while i was just navigating the menus.

Im on Windows 10 with an AMD Radeon RX 5700XT and an AMD Ryzen 7 3700X processor.
Im Using opentoonz 1.4.0

**Comment by jrhyme** [comment]
According to such a report, I can only answer - but nothing breaks at all on all operating systems, it works stably. Where are the video errors? where, as a last resort, a log file.

**Comment by jrhyme** [comment]
> According to such a report, I can only answer - but nothing breaks at all on all operating systems, it works stably. Where are th errors? record a video, where, as a last resort, a log file.

**Comment by M1ntGr33n** [comment]
> According to such a report, I can only answer - I have nothing breaks at all on all operating systems, it works stably. Where are the errors? record a video, upload a log file.

where can i find the log files?

**Comment by RodneyBaker** [comment]
For most general stability issues I tend to recommend backing up the current stuff directory and reinstalling.  This gives Opentoonz a clean set of instructions (primarily for the UI) to run.  This is especially useful in two cases:  Where an update is being made and when it has been a long time since the stuff directory has been cleaned out/refreshed.

Also related, Opentoonz generally does not like a lot a files in the same directory.
I haven't investigated to see what the threshold is but in general any program will run better if it doesn't have to parse through a lot of files... and Opentoonz tries to keep an updated listing of all files (and which files are parts of sequences) in any active directory.

**Comment by manongjohn** [comment]
Are you able to create any levels and save?

Though it doesn't always help, you can look at Windows Event for an Application log that shows why OT crashed. Paste it here and we can see if it gives any idea what's going on.

**Comment by M1ntGr33n** [comment]
> 
> 
> Are you able to create any levels and save?
> 
> Though it doesn't always help, you can look at Windows Event for an Application log that shows why OT crashed. Paste it here and we can see if it gives any idea what's going on.

yeah im able to save and everything, besides the crashes it works fine, i even can render if i manage to do that before the program crashes.
And the thing with the event viewer is that there is no crash log. its like opentoonz didnt crash but just got closed normalyy. thats the wierd part. are there any opentoonz logs in the opentoonz files somewhere?

**Comment by manongjohn** [comment]
Hmm. When OT closes like that for me I usually see an application log in the event viewer. I think maybe only once I didn't. Unfortunately there are no other logs.

If 1.4 is giving you a lot of problems, I wonder if dropping back to 1.3 will help. If you've not used 1.3 before and you find it more stable, it could show it's something new in 1.4 might be an issue on your system. Btw, I'm assuming you are using the Official 1.4 and not the Morevna version.

**Comment by M1ntGr33n** [comment]
> 
> 
> Hmm. When OT closes like that for me I usually see an application log in the event viewer. I think maybe only once I didn't. Unfortunately there are no other logs.
> 
> If 1.4 is giving you a lot of problems, I wonder if dropping back to 1.3 will help. If you've not used 1.3 before and you find it more stable, it could show it's something new in 1.4 might be an issue on your system. Btw, I'm assuming you are using the Official 1.4 and not the Morevna version.


Ok so i tested a bit and 1.3 is the only version that works. Nothing before and nothing after.
I really cant figure out what is happening here

**Comment by manongjohn** [comment]
Do you have any antivirus software running? Try turning it off temporarily and see if somehow it keeps killing it?

**Comment by andeon** [comment]
@RodneyBaker This reported issue technically does not provide useful information to identify the problem, so I recommend closing it. The user could test the [nightly version](https://github.com/opentoonz/opentoonz/releases) and, if the issue persists, open a new report, since this concerns an old OpenToonz version.
